packages/containers/psammead-timestamp-container/src/index.jsx

```diff
@@ -55,6 +60,7 @@ TimestampContainer.defaultProps = {
   padding: true,
   prefix: null,
   suffix: null,
+  locale: 'en',
```

Comment:
I think it should default to null, as we use `en-gb` in psammead so it defaults to whatevers availible. i think if you give it `en-gb` but ask for `en`, it will use `en-gb` but throw a warning.